src/transformers/activations_tf.py

```diff
@@ -56,14 +57,20 @@ def mish(x):
 
 def gelu_fast(x):
     x = tf.convert_to_tensor(x)
-    coeff1 = tf.cast(7978845608, x.dtype)
+    coeff1 = tf.cast(0.7978845608, x.dtype)
```

Comment:
wow was that wrong the whole time before?

Reply:
Yep! I'm surprised that this had not been spotted before 😄